go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/codeskyblue/go-sh v0.0.0-20200712050446-30169cf553fe
 	github.com/go-logr/logr v1.2.3
 	github.com/go-logr/zapr v1.2.3
+	github.com/kedacore/keda/v2 v2.9.3
```

Comment:
Does registering versions relations make sense @tomkerkhove ?
I guess that this dep works also with newer code because we haven't introduced any change here, but maybe having a table relating HTTP Add-on version with KEDA versions could be nice. 
At least with KEDA main and KEDA -1.
WDYT @tomkerkhove ?
We should introduce multiple KEDA versions for the e2e tests too (just editing a bit the execution script to allow use arguments for setting KEDA version)

Reply:
I don't have a preference here

Reply:
I'd create a compatibility table in docs to list these relations something like:
|HTTP-AddOn|KEDA|Kubernetes|
|---------------|-----|-------------|
| 0.4.0|v2.x-v2.z|1.xx-1.zz|

At least with known version, like we do in [core docs](https://keda.sh/docs/latest/operate/cluster/), basically I'd like to avoid issue about incompatibilities, or at least a documented section to point them.
Could you add something like this to docs (maybe at the end of [install section](https://github.com/kedacore/http-add-on/blob/main/docs/install.md))

Reply:
Done!